src/main/java/me/ncbpfluffybear/fluffymachines/FluffyItemSetup.java

```diff
@@ -24,86 +25,102 @@
 public final class FluffyItemSetup {
 
     private FluffyItemSetup() {}
+    private static final SlimefunItemStack circuitBoard = SlimefunItems.ADVANCED_CIRCUIT_BOARD;
+    private static final ItemStack orangeGlass = new ItemStack(Material.ORANGE_STAINED_GLASS);
+    private static final ItemStack brownGlass = new ItemStack(Material.BROWN_STAINED_GLASS);
 
     public static void setup(@Nonnull FluffyMachines plugin) {
-
-        new WateringCan(FluffyItems.fluffymachines, FluffyItems.WATERING_CAN, RecipeType.ENHANCED_CRAFTING_TABLE, new ItemStack[] {
+        
+        new WateringCan(FluffyItems.fluffymachines, FluffyItems.WATERING_CAN, RecipeType.ENHANCED_CRAFTING_TABLE,
+            new ItemStack[] {
```

Comment:
Sometimes you put the recipe type on this line, other times you dont. maybe better if you did